metadata-ingestion/scripts/install_deps.sh

```diff
@@ -4,10 +4,23 @@ set -euxo pipefail
 if [ "$(uname)" == "Darwin" ]; then
     brew install librdkafka
 else
-    sudo apt-get update && sudo apt-get install -y \
-        librdkafka-dev \
-        python3-ldap \
-        libldap2-dev \
-        libsasl2-dev \
-        ldap-utils
+    sudo_cmd=""
+    if command -v sudo; then
+        sudo_cmd="sudo"
+    fi
```

Comment:
Not sure if we care, but this will print the result of `command -v sudo` to stdout. Can either pipe into /dev/null or check `[ -x "$(command -v sudo)" ]` as per https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script